@ecomplus/storefront-components/src/html/CartQuickview.html

```diff
@@ -102,6 +102,21 @@
           >
             {{ i19seeCart }}
           </a-link>
+
+          <button
+            type="button"
+            class="minicart__btn-cart btn btn-block btn-outline-secondary close d-md-none"
+            :aria-label="i19close"
+            @click="toggle"
+          >
+            <small
+              aria-hidden="true"
+            >
+              <i class="fas fa-arrow-left mr-1"></i>
+              {{ i19continueShopping }}
+            </small>
```

Comment:
```suggestion
            <i class="fas fa-arrow-left mr-1"></i>
            {{ i19continueShopping }}
```

Na verdade não entendi o porquê desse elemento `<small aria-hidden>`, se a intenção era só reduzir a fonte do botão o certo é usar `btn-sm`